packages/react-sdk/docusaurus/docs/React/02-tutorials/02-audio-room.mdx

```diff
@@ -3,4 +3,543 @@ title: Audio Room Tutorial
+:::
+
+Alright, let's get started. Creating this app will require the following steps:
+
+1. Setup the project
+2. Describe the login flow
+3. Creating rooms via the API
+4. Querying calls (with real-time updates)
+5. Joining and leaving a room
+6. Building up a UI for a room
+7. Managing speaking requests
+
+### Setup the project
+
+The first step is to set up a new project. We'll use `vite` to do this, but feel free to use any other tool that you like.
+
+To keep things simple, we'll just work with plain React and Typescript:
+
+```bash
+yarn create vite audio-rooms-sample --template react-ts
+```
+
+:::note
+The entire Stream team loves Typescript. If you're wondering why, consider checking [this post](https://getstream.io/blog/typescript-basics/) about the topic on our Blog.
+:::
+
+Next, we install the dependencies for the Stream package:
+
+```bash
+yarn add @stream-io/video-react-sdk
+```
+
+After that, we can create a new project in the [Stream Dashboard](https://getstream.io/dashboard/). Once logged in, you will be introduced to our dashboard. There you can manage your applications. The app represents Stream's dedicated infrastructure that will handle your app's video calls.
+Make sure to [create a video application](TODO: image/gif showing the UI creating the app) in your Stream dashboard. The API key and secret will be generated automatically upon the app's creation. Those are the credentials we will need for authentication.
+
+:::tip
+If you don't have a Stream account yet, you can register and try it for free [here](https://getstream.io/try-for-free/).
+:::
+
+The last setup step is to create a `.env` file. This is where we will put our API key to keep it safe.
+Create a `.env` file in the root of your project and paste this code in (and replace `<your_api_key>` with the real one):
+
+```
+VITE_STREAM_API_KEY=<your_api_key>
+```
+
+With that the basic setup of the project and the Stream SDK is ready and we can dive into the code.
+
+### User Authentication
+
+We won't cover the entire flow of authentication in this guide because we want to focus on the important parts.
+You can check the complete flow in the sample app repository, and we'll go through the important API calls now.
+
+The first important step is to initialize the Stream client. Therefore, we need the API key and a token provider.
+
+:::tip
+Want to learn why we need the token provider and how it works? See [this page](../../guides/client-auth/#token-providers) for more detail.
+:::
+
+First, we can collect the API key from our `.env` file by simply importing it like this:
+
+```
+const apiKey = import.meta.env.VITE_STREAM_API_KEY as string;
+```
+
+Next, to get a token, we need to create one in our backend. Whenever this expires the SDK will call the token provider again to renew the token.
```

Comment:
I think that part is already covered in the Authentication guide, so linking to that seems enough for me, but if we leave that part in, we should mention the [token generator](https://staging.getstream.io/video/docs/react/guides/client-auth/#introduction) as well, as probably most users won't have their token generator endpoint ready by the time they are doing this tutorial